src/System.Runtime.Serialization.Xml/tests/DataContractSerializer.cs

```diff
@@ -1703,6 +1703,17 @@ public static void DCS_DerivedException()
         Assert.StrictEqual(value.TestProperty, actual.TestProperty);
     }
 
+    [Fact]
+    public static void DCS_DeserializeEmptyString()
+    {
+        var serializer = new DataContractSerializer(typeof (object));
+        var exception = Assert.Throws<XmlException>(() =>
+        {
+            serializer.ReadObject(new MemoryStream());
+        });
+        Assert.StrictEqual("Unexpected end of file.", exception.Message);
```

Comment:
The issue is there are differences in the message and location of the XmlException in NetCore compared to Desktop. So I want to ensure that this scenario throws an XmlException that is not too different from Desktop.